### PR TITLE
docu: Correct disabled device for http adapter

### PR DESCRIPTION
**Pull request by pho-enix**
It is actually 404 NOT FOUND

Signed-off-by: Sebastian Poehn <sebastian.poehn@bosch-si.com>

### CODE REVIEW

File: site/content/user-guide/http-adapter.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Publish some JSON data for device `4711`, indicating that the device will wait f
         * The tenant that the gateway belongs to is not allowed to use this protocol adapter.
         * The device belongs to another tenant than the gateway.
         * The gateway is not authorized to act *on behalf of* the device.
-        * The gateway is disabled.
+  * 404 (Not Found): The request cannot be processed because the device is disabled or does not exist.
```
Comment (sophokles73) [inline site/content/user-guide/http-adapter.md]: have you actually tried the case of a disabled gateway? According to the Device Registration API this should indeed result in a 403. However, adding the 404 and its description is still correct...

File: site/content/user-guide/http-adapter.md [hunk 2]
```diff
@@ -330,7 +330,7 @@ Publish some JSON data for device `4711`:
...
```
Comment (sophokles73) [inline site/content/user-guide/http-adapter.md]: see above

File: site/content/user-guide/http-adapter.md [hunk 3]
```diff
@@ -475,7 +475,7 @@ Send a response to a previously received command with the command-request-id `re
   * 403 (Forbidden): The request cannot be processed because the device's registration status cannot be asserted. Possible reasons for this might be:
         * The given tenant is not allowed to use this protocol adapter.
         * The given device does not belong to the given tenant.
-        * The given device is disabled.
+  * 404 (Not Found): The request cannot be processed because the device is disabled or does not exist.
```
Comment (sophokles73) [inline site/content/user-guide/http-adapter.md]: see above

File: site/content/user-guide/http-adapter.md (hunk 1 above)
Comment (pho-enix) [reply site/content/user-guide/http-adapter.md]: No, I didn't check this with gateways.

Comment (sophokles73) [reply site/content/user-guide/http-adapter.md]: Can you then please update the PR to keep the case of a disabled gateway in the list for the 403 status code? And please  don't forget the other two places where this applies as well ...

Comment (pho-enix) [reply site/content/user-guide/http-adapter.md]: For gateway mode:
Gateway is disabled: 403
Device is disabled: 404

Changed PR accordingly.

**Pull request merged**